### DISCUSSION TITLE
Support additional domain-specific visualizations

### DISCUSSION THREAD

**Discussion by bilalshaikh42** [biosimulations/biosimulations#2775] [Ideas]
@jonrkarr 
This may be an interesting use case/related work  that we can look at for our discussions on visualizations and published figures.  Have just skimmed, but seems like it has a declarative format for specifying at least part of the figure. 

https://github.com/deeptools/pyGenomeTracks

**Comment by jonrkarr** [comment]
There's many other formats that could be supported
- Plotly
- ggplot2 --> plotly (via ggplotly)
- matplotlib/seaborn scripts
- Vtu files (for use with Vtk)

Once we're more satisfied with core features, I think we should try to survey users to help prioritize further efforts. For example, while all of these formats could be useful in certain cases, its not obvious to me which of these, if any, are popular among investigators doing simulations. A survey could help us focus on features that would impact users.

**Comment by bilalshaikh42** [comment]
Agreed. I filed this issue as a reminder regarding the list of scientific visualization types that we were interested in thinking of as case studies for visualization
